src/stackmap_checker/stmap.c

```diff
@@ -75,7 +75,6 @@ stack_map_record_t* stmap_get_map_record_in_func(stack_map_t *sm,
                                                  uint64_t patchpoint_id,
                                                  uint64_t fun_addr)
 {
-    fprintf(stderr, "looking for %d in %p\n", patchpoint_id, fun_addr);
```

Comment:
Ah, I see, it was killed in a later commit. Perhaps try to ensure they don't make it into commits in future PRs.